sql/mob_spawn_points.sql

```diff
@@ -6972,14 +6972,14 @@ INSERT INTO `mob_spawn_points` VALUES (16916551,'Eozdei','Eo\'zdei',2,-780.000,-
 INSERT INTO `mob_spawn_points` VALUES (16916552,'Eoghrah','Eo\'ghrah',1,-740.135,-0.500,516.776,200);
 INSERT INTO `mob_spawn_points` VALUES (16916553,'Eozdei','Eo\'zdei',2,-700.000,-0.500,500.000,176);
 INSERT INTO `mob_spawn_points` VALUES (16916554,'Eozdei','Eo\'zdei',2,-700.000,-0.500,420.000,160);
-INSERT INTO `mob_spawn_points` VALUES (16916555,'Eoeuvhi','Eo\'euvhi',3,-740.000,-0.500,492.000,192);
-INSERT INTO `mob_spawn_points` VALUES (16916556,'Eoeuvhi','Eo\'euvhi',3,-740.000,-0.500,486.000,192);
+INSERT INTO `mob_spawn_points` VALUES (16916555,'Eoeuvhi_open','Eo\'euvhi',3,-740.000,-0.500,492.000,192);
+INSERT INTO `mob_spawn_points` VALUES (16916556,'Eoeuvhi_open','Eo\'euvhi',3,-740.000,-0.500,486.000,192);
 INSERT INTO `mob_spawn_points` VALUES (16916557,'Eoeuvhi','Eo\'euvhi',3,-712.685,-0.504,460.449,24);
-INSERT INTO `mob_spawn_points` VALUES (16916558,'Eoeuvhi','Eo\'euvhi',3,-740.000,-0.500,428.000,64);
-INSERT INTO `mob_spawn_points` VALUES (16916559,'Eoeuvhi','Eo\'euvhi',3,-740.000,-0.500,434.000,64);
+INSERT INTO `mob_spawn_points` VALUES (16916558,'Eoeuvhi_open','Eo\'euvhi',3,-740.000,-0.500,428.000,64);
+INSERT INTO `mob_spawn_points` VALUES (16916559,'Eoeuvhi_open','Eo\'euvhi',3,-740.000,-0.500,434.000,64);
 INSERT INTO `mob_spawn_points` VALUES (16916560,'Eozdei','Eo\'zdei',2,-797.000,-1.500,460.000,0);
 INSERT INTO `mob_spawn_points` VALUES (16916561,'Eoghrah','Eo\'ghrah',1,-657.978,-0.606,458.174,19);
-INSERT INTO `mob_spawn_points` VALUES (16916562,'Eoghrah','Eo\'ghrah',1,-658.148,-0.621,461.730,253);
+INSERT INTO `mob_spawn_points` VALUES (16916562,'Eoghrah','Eo\'ghrah',1,-620.467,0.000,460.072111);
```

Comment:
Error Here:

 INSERT INTO `mob_spawn_points` VALUES (16916562,'Eoghrah','Eo\'ghrah',1,-620.467,0.000,460.072111);
 
 Your Z pos and Rotation got merged into one value

Reply:
Corrected